parser_library/test/processing/copy_mock.h

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2019 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+#ifndef HLASMPLUGIN_PARSERLIBRARY_TEST_COPY_MOCK_H
+#define HLASMPLUGIN_PARSERLIBRARY_TEST_COPY_MOCK_H
+
+#include "../mock_parse_lib_provider.h"
+#include "analyzer.h"
+
+namespace hlasm_plugin::parser_library {
+
+class copy_mock
+{
+public:
+    static mock_parse_lib_provider create()
```

Comment:
move to cpp file?

Reply:
the class does not seem to be used for anything anymore. Should we have just create_copy_mock() standalone function then?